packages/types/src/index.ts

```diff
@@ -389,6 +389,20 @@ export interface MessageAttachment {
   actions?: AttachmentAction[];
   callback_id?: string;
   mrkdwn_in?: ('pretext' | 'text' | 'fields')[];
+  app_unfurl_url?: string;
+  is_app_unfurl?: boolean;
+  app_id?: string;
+  bot_id?: string;
+  preview?: MessageAttachmentPreview; // https://api.slack.com/methods/chat.unfurl#markdown
+}
+
+// https://api.slack.com/methods/chat.unfurl#markdown
+export interface MessageAttachmentPreview {
+  type?: string;
+  can_remove?: boolean;
+  title?: PlainTextElement;
+  subtitle?: PlainTextElement;
+  iconUrl?: string;
```

Comment:
Are all of these actually optional? I know nothing of the payload, but it strikes me as odd to have something like `type` potentially not be present.

Reply:
Since I haven't checked all the patterns, I've marked all the properties as optional for safety